### ISSUE TITLE
Should enter key copy block's alignment?

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-alignment#8]
Should this be the <kbd>Enter</kbd> key's default behaviour or something implemented by the ckeditor5-alignment package?

Related ticket: https://github.com/ckeditor/ckeditor5-enter/issues/40.

**Comment by jodator** [comment]
Thinking about wheter it should copy or not it heavily depends on context. It's either you want to have alignment to be copied or not.

Case 1: Writing letters or papers - when you format recipient address to be on the right and then hit `enter` and you'd like to write header and make it centered. Then `enter` again and you start paragraph that should be aligned normally (to the left).

Case 2: You write a blog post with song lyrics. So you hit `enter` change alignment to center and you keep writing song lyrics after. Here each `enter` will should keep alignment. When you done with lyrics you change alignment to default.

I'd go with anything that is standard in text processors here. I suspect that "copy" is the default one. 

AFAIR you can define styles in Word/LibreOffice document in a way that you can define what type of paragraph (style) is a following (ie after header of type A you have paragraph B with different alignment, font, etc) - but this looks like a pro user setting.

ps.: I've just checked LibreOffice and the alignment is kept on `enter` (new paragraph).

ps.: the mentioned setting in LibreOffice for styles (Next style):
![selection_026](https://user-images.githubusercontent.com/247363/32948695-72905bd8-cba0-11e7-8e77-a40413caa0d1.png)


TL;DR: I'd co with copy alignment on enter.

**Comment by Reinmar** [comment]
This has been changed in the recent release – inline styles such as bold and italic as well as block styles such as alignment are now copied to the next line.

**Comment by Reinmar** [comment]
If anyone wants to turn this behaviour it can be done by using https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_model_schema-Schema.html#function-setAttributeProperties (and setting `copyOnEnter` to `false` for all styles).
